### DISCUSSION TITLE
npmjs.com CSS bug

### DISCUSSION THREAD

**Discussion by LookRain** [npm/feedback#720] [Registry] [answered]
Text are not properly wrapped on https://www.npmjs.com/package/sm2
![image](https://user-images.githubusercontent.com/11829847/176652880-016f90f1-2ee7-4836-92d6-73f6c755c38e.png)

Potential Solution
![image](https://user-images.githubusercontent.com/11829847/176652950-2ad2c343-d66f-4022-a9a6-4ded0b500b15.png)

adding a css property seems to fix it, but not sure if that's the best solution to it

**Comment by monishcm** [comment, marked as answer]
Thanks for the feedback. We will add this to our backlog.
